common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -54,12 +55,14 @@ public class CommonValue {
     public static final String ML_MODEL_INDEX = ".plugins-ml-model";
     public static final String ML_TASK_INDEX = ".plugins-ml-task";
     public static final Integer ML_MODEL_GROUP_INDEX_SCHEMA_VERSION = 2;
-    public static final Integer ML_MODEL_INDEX_SCHEMA_VERSION = 8;
+    public static final Integer ML_MODEL_INDEX_SCHEMA_VERSION = 10;
     public static final String ML_CONNECTOR_INDEX = ".plugins-ml-connector";
     public static final Integer ML_TASK_INDEX_SCHEMA_VERSION = 2;
     public static final Integer ML_CONNECTOR_SCHEMA_VERSION = 2;
     public static final String ML_CONFIG_INDEX = ".plugins-ml-config";
     public static final Integer ML_CONFIG_INDEX_SCHEMA_VERSION = 2;
+    public static final String ML_MODEL_CONTROLLER_INDEX = ".plugins-ml-model-controller";
```

Comment:
Our goal here is to use a name that is general enough so that in the future we won't build another index. This has been discussed in our design review meeting.

Reply:
For general purpose how about `.plugins-ml-controller` ? As we have agent and connector, they may also need controller

Reply:
That's a good point. I agree with this. In addition, I feel like controller is a wide term? What kind of control this controller should have? Like we have model access control. Is this usage control? Or any kind of control in general?

Reply:
Sounds like a good name to me, I agree with `.plugins-ml-controller`